flocker/testtools/_base.py

```diff
@@ -104,6 +128,112 @@ def mktemp(self):
         return make_temporary_directory(self).child('temp').path
 
 
+def _fix_twisted_logs(log_content):
+    """
+    Split the Eliot logs out of a Twisted log.
+
+    :param Content log_content: A text content object that contains a Twisted
+        log.
+    :return: The log split into two, the first containing the core Twisted log
+        messages and the second containing line-separated Eliot JSON messages.
+    :rtype: (Content, Content)
+    """
+    twisted_lines, eliot_lines = tee(_iter_content_lines(log_content))
+    return (
+        Content(UTF8_TEXT, partial(_pure_twisted_logs, twisted_lines)),
+        Content(
+            UTF8_TEXT,
+            lambda: _prettyformat_lines(_iter_eliot_logs(eliot_lines)),
+        ),
+    )
+
+
+def _pure_twisted_logs(log_lines):
+    """
+    Given an iterable of lines of bytes of Twisted logs, yield the lines that
+    are not Eliot data.
+    """
+    return (line for line in log_lines if _get_eliot_data(line) is None)
+
+
+def _prettyformat_lines(lines):
+    """
+    Pretty format lines of Eliot logs.
+    """
+    for line in lines:
+        data = json.loads(line)
+        yield pretty_format(data)
```

Comment:
`eliot-prettyprint` puts a blank line between stanzas. Is it worth doing that here?